test/e2e/specs/editor/plugins/iframed-masonry-block.spec.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'iframed masonry block', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-iframed-masonry-block'
+		);
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-iframed-masonry-block'
+		);
+	} );
+
+	test( 'should load script and dependencies in iframe', async ( {
+		editor,
+	} ) => {
+		await editor.insertBlock( { name: 'test/iframed-masonry-block' } );
+
+		const masonry = editor.canvas.getByRole( 'document', {
+			name: 'Block: Iframed Masonry Block',
+		} );
+		await expect( masonry ).toBeVisible();
```

Comment:
I've replaced the snapshot assertion with this. Do we need to assert the block's whole edit markup?

Reply:
I don't think we do. If we want anything tested then we should probably add the assertions below anyway.